### DISCUSSION TITLE
Feature Request: why there is no tools/rpkg  now?

### DISCUSSION THREAD

**Discussion by longhua8800w** [duckdb/duckdb#13132] [Ideas]
I want to use duckdb to handle json parsing, I have to add extensions so I follw this instructions https://github.com/duckdb/duckdb-r and clone https://github.com/duckdb/duckdb.git   and cd "duckdb/
EXTENSION_STATIC_BUILD=1 make"  seems  everything is ok,  build sucessed.  but i can't find path "tools/rpkg "  , so i can't use R CMD INSTALL .  to create a R package   with extentions.    
I have search for similar discussions, only to find that there should be  path "tools/rpkg " .
how can i sovle this ?

**Comment by Tishj** [comment]
It was moved to the `duckdb-r` repo, I assume the README is not updated to reflect that?
It no longer lives in core: <https://github.com/duckdb/duckdb/pull/8793>

**Reply by longhua8800w** [reply]
this instructions https://github.com/duckdb/duckdb-r  seems too old because it still lead me to  https://github.com/duckdb/duckdb.git ; and there seems no way to add extensions  to duckdb-r.

**Comment by longhua8800w** [comment]
I find that duckdb has not support  json extension now; only support parquet
![image](https://github.com/user-attachments/assets/2fbd8da9-57cb-4099-acbf-8a433c44ec13)

**Reply by Tishj** [reply]
Not supporting and not having the extension built in are two different things, I am confident you can still install+load the JSON extension in the R package

**Reply by longhua8800w** [reply]
Thanks for the tip, I'll keep trying to find a solution.
